docs/book/go_docs.md

```diff
@@ -0,0 +1,24 @@
+## GoDoc
+
+Many of the kubebuilder libraries can be used on their own without the kubebuilder code generation and scaffolding.
+
+See examples of using the libraries directly below:
+
+- [controller libraries](https://godoc.org/github.com/kubernetes-sigs/kubebuilder/pkg/controller)
+- [config libraries](https://godoc.org/github.com/kubernetes-sigs/kubebuilder/pkg/config)
+- [signals libraries](https://godoc.org/github.com/kubernetes-sigs/kubebuilder/pkg/signals)
+
+Kubebuilder generates codes for custom resource fields, and controller components such as watchers and informers. You have to add code generation tags in form of comment directives to initiate the code generation:
```

Comment:
These code generation tags are so important. I missed one of the informer tag in the core type controller template. It caused a failure and I spent a few hours debugging it. I'd like to see we put the docs for the tags in a more obvious place. Something like `Important note` in the controller documentation.

Reply:
There is a section on this in the Controller section.  Added a `warning` note as well.